vite.config.ts

```diff
@@ -36,13 +30,13 @@ export default defineConfig({
       // eslint-disable-next-line unicorn/prefer-module
       entry: path.resolve(__dirname, 'src/main.ts'),
       name: 'Prime',
-      fileName: (format) => `prime.${format}.js`,
```

Comment:
I set this back to vite defaults to ensure the UMD build gets a `.cjs` extension, which is important because our `package.json` spec's `type: module`, so the `cjs` is important to opt-out of module behavior